src/commands/rate.ts

```diff
@@ -0,0 +1,74 @@
+import { LanguageKeys } from '#lib/i18n/LanguageKeys';
+import { escapeMarkdown } from '@discordjs/builders';
+import { Command, RegisterCommand } from '@skyra/http-framework';
+import { applyLocalizedBuilder, getT } from '@skyra/http-framework-i18n';
+
+const Root = LanguageKeys.Commands.Rate;
+const OWNERS = ['242043489611808769', '268792781713965056'];
+const ONE_TO_TEN = new Map<number, UtilOneToTenEntry>([
```

Comment:
Move these to `private readonly` and camelCase the names. The map can be `oneToTenMap`.